src/main/java/org/webjars/WebJarVersionLocator.java

```diff
@@ -126,6 +134,37 @@ public String version(final String webJarName) {
         return optionalVersion.orElse(null);
     }
 
+    private void readLocatorProperties() {
+        try {
+            Enumeration<URL> resources =
+                LOADER.getResources(WEBJARS_PATH_PREFIX + LOCATOR_PROPERTIES);
+            while (resources.hasMoreElements()) {
+                URL resourceUrl = resources.nextElement();
+                try (InputStream resource = resourceUrl.openStream()) {
+                    Properties properties = new Properties();
+                    properties.load(resource);
+                    for (Map.Entry<Object, Object> entry : properties.entrySet()) {
+                        String webJarName = entry.getKey().toString();
+                        if (!webJarName.endsWith(".version")) {
+                            // ".version" suffix is required
+                            continue;
+                        }
+
+                        webJarName = webJarName.substring(0, webJarName.lastIndexOf(".version"));
+
+                        String version = entry.getValue().toString();
+                        if (hasResourcePath(webJarName, version)) {
+                            // Only add configured versions if their path exists
+                            cache.computeIfAbsent(CACHE_KEY_PREFIX + webJarName, x -> Optional.of(version));
+                        }
+                    }
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("unable to load locator properties", e);
```

Comment:
Not sure if this exception should be rethrown, logged or just ignored (similar to `version()`). For now I opted for throwing.